src/async_block_store.rs

```diff
@@ -145,7 +150,59 @@ impl<U: Unblocker> AsyncBlockStore<U> {
         f: impl FnOnce(&mut BlockStore) -> T + Send + 'static,
     ) -> BoxFuture<T> {
         let inner = self.inner.clone();
-        self.unblocker
+        self.runtime
             .unblock(move || f(DerefMut::deref_mut(&mut inner.lock().unwrap())))
     }
+
+    /// A gc loop that runs incremental gc in regular intervals
+    ///
+    /// Gc will run as long as this future is polled.
```

Comment:
* Maybe add a note about running `incremental_delete_orphaned` after `interval / 2`.
* Also, maybe don't start gc'ing right away, but first wait `interval / 2`.
* What about adding some tracing span instrumentation here?

Reply:
Good points.

Will add the initial delay and add some notes.

The incremental_x methods themselves do some timing. I did not use tracing spans there since I want the log level to change depending on if it is within a time budget or not, and tracing spans can't do that.